src/actions/walletConnectActions.js

```diff
@@ -399,11 +426,15 @@ export const approveSessionAction = (peerId: string) => {
     if (peerMeta) {
       dispatch(killWalletConnectSessionByUrl(peerMeta.url, peerId));
     }
-    const { wallet: { data: { address } } } = getState();
-
+    const {
+      accounts: { data: accounts },
+    } = getState();
     try {
+      const smartAcc = findFirstSmartAccount(accounts);
```

Comment:
This should be first Smart Wallet account or just current active Smart Wallet account?

Reply:
Can we have multiple Smart Wallet accounts? 👀

Reply:
Technically we can since SW sdk returns us an array of account we have, but we don't allow user via our app to create more than one account